src/vmm/src/builder.rs

```diff
@@ -1511,14 +1468,7 @@ pub mod tests {
             stats_polling_interval_s: 0,
         };
 
-        let mut cmdline = default_kernel_cmdline();
-        insert_balloon_device(&mut vmm, &mut cmdline, &mut event_manager, balloon_config);
-        // Check if the vsock device is described in kernel_cmdline.
-        #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
-        assert!(cmdline_contains(
-            &cmdline,
-            "virtio_mmio.device=4K@0xd0000000:5"
-        ));
+        insert_balloon_device(&mut vmm, &mut event_manager, balloon_config);
```

Comment:
This test does not actually assert anything anymore now, so I think we can remove it altogether

Reply:
Oh, fair, didnt see that. Nvm then, sorry!

Reply:
no worries! Hadn't seen it myself up until 15' ago :P